### DISCUSSION TITLE
Does "next export" support next.config.js rewrites?

### DISCUSSION THREAD

**Discussion by rayventerprise** [vercel/next.js#24627] [Help]
Hey all,

I am trying to export my Next.js project to an Android app using Capacitor. It so far is working, however the last step is to get my rewrites to work. So currently I have this in my next.config.js.

```
async rewrites() {
    return [
        {
            source: '/api/:slug*',
            destination: `${process.env.NEXT_PUBLIC_API_DOMAIN}api/:slug*`
        }
    ]
}
```

I have to use "next export" for Capacitor to wrap my next project into an Android app which works. However I don't believe "next export" is properly rewriting the URLs based upon my rewrites. What should I do here? I am at the point where my app loads on an Android simulator, however none of the requests will load due to the route not existing since theres no server for it to proxy against.

**Comment by cyntler** [comment]
Any updates?

**Reply by tomaszstrejczek** [reply]
According to https://nextjs.org/docs/advanced-features/static-html-export, rewrites are not supported by next export.
